assets/js/modules/analytics/components/settings/GA4SettingsControls.js

```diff
@@ -247,21 +253,23 @@ export default function GA4SettingsControls( {
 						outlined
 					>
 						<Option value={ matchedProperty?._id || '' }>
-							{ ! matchedProperty?._id ||
-							! matchedProperty?.displayName
-								? ''
-								: sprintf(
-										/* translators: 1: Property name. 2: Property ID. */
-										_x(
-											'%1$s (%2$s)',
-											'Analytics property name and ID',
-											'google-site-kit'
-										),
-										matchedProperty.displayName,
-										measurementIDs?.[
-											matchedProperty._id
-										] || ''
-								  ) }
+							{ ! matchedProperty?._id && '' }
+
+							{ matchedProperty?._id &&
+								( measurementIDs?.[ matchedProperty._id ]
+									? sprintf(
+											/* translators: 1: Property name. 2: Property ID. */
+											_x(
+												'%1$s (%2$s)',
+												'Analytics property name and ID',
+												'google-site-kit'
+											),
+											matchedProperty.displayName,
+											measurementIDs?.[
+												matchedProperty._id
+											] || ''
```

Comment:
I think this conditional chain and fallback `''` can be removed since it's already checked at the top?

Reply:
Ah, should've noticed it earlier. Thank you for pointing it out! Updated.